ddtrace/llmobs/_llmobs.py

```diff
@@ -120,6 +120,7 @@ def __init__(self, tracer=None):
 
     def _on_span_start(self, span):
         if self.enabled and span.span_type == SpanTypes.LLM:
+            self._activate_llmobs_span(span)
```

Comment:
by moving this here, are we double activating with what we already have in `_start_span`? (and does that matter - wonder if it's the case for the failing tests) https://github.com/DataDog/dd-trace-py/blob/90e161691323feb2436ae987194825d969213205/ddtrace/llmobs/_llmobs.py#L616

that line happens after `self.tracer.trace(...)`, which should call the `_on_span_start` hook, and then activate again, for manually instrumented spans

Reply:
I replaced the activation call in `_start_span()` with it here in `_on_span_start()` so don't think this should be an issue, but looking into the failing tests to verify!

Reply:
if so i guess we can remove the activation from `_start_span`?

Reply:
Think I've done it already? https://github.com/DataDog/dd-trace-py/pull/12615/files#r1978327936

Reply:
oh i also totally just misread the diff lol. disregard